src/backend/expungeservice/models/charge_classifier.py

```diff
@@ -109,7 +109,7 @@ def _parking_ticket(statute, chapter):
 
     @staticmethod
     def _schedule_1_pcs(section):
-        if section in ['475854', '475874', '475884', '475894']:
+        if section in ['475854', '475874', '475884', '475894'] or section[:6] == '475992':
```

Comment:
```suggestion
        if section[:6] in ['475854', '475874', '475884', '475894', '475992']
```

Reply:
I considered that but I wanted to be super cautious about changing the legal logic here. Your suggestion would cause something like `4758544A` to be classified as PCS too. That's probably fine though, because the subsequent digits are subsections.

Reply:
If you're worried, I would add a comment that you're not sure. Otherwise it looks like the subsections actually matter for the previous 4.